cmd/subnetcmd/join.go

```diff
@@ -360,6 +437,48 @@ func checkIsValidating(subnetID ids.ID, nodeID ids.NodeID, pClient platformvm.Cl
 	return false, nil
 }
 
+func promptNodeIDToAdd() (ids.NodeID, error) {
+	if nodeIDStr == "" {
+		ux.Logger.PrintToUser("Please enter the NodeID of the validator that you want to join this elastic subnet.")
+		ux.Logger.PrintToUser("")
+		ux.Logger.PrintToUser("Check https://docs.avax.network/apis/avalanchego/apis/info#infogetnodeid for instructions about how to query the NodeID from your node")
+		promptStr := "What is the NodeID of the validator you'd like to join this elastic subnet?"
+		nodeID, err := app.Prompt.CaptureNodeID(promptStr)
+		if err != nil {
+			return ids.NodeID{}, err
+		}
+		return nodeID, nil
+	}
+	nodeID, err := ids.NodeIDFromString(nodeIDStr)
+	if err != nil {
+		return ids.NodeID{}, err
+	}
+	return nodeID, nil
+}
+
+func promptStakeAmount() (uint64, error) {
+	if stakeAmount > 0 {
+		return stakeAmount, nil
+	}
+	defaultWeight := fmt.Sprintf("Default (%d)", constants.DefaultPermissionlessSubnetValidatorStake)
```

Comment:
How are you setting this default? I think we probably want it to be the full amount they have (`max`)